src/main/java/com/gooddata/executeafm/afm/PopMeasureDefinition.java

```diff
@@ -41,36 +39,17 @@
      *         measure identifier
      * @param popAttribute
      *         "period over period" date attribute
-     * @param offset
-     *         the number of periods the time window defined by {@code popAttribute} is offset about to the past (when value is negative) or future (when
-     *         value is positive)
      */
     @JsonCreator
     public PopMeasureDefinition(@JsonProperty("measureIdentifier") final String measureIdentifier,
-                                @JsonProperty("popAttribute") final ObjQualifier popAttribute,
```

Comment:
it's backward incompatible change - pls rather make this constrcutor deprecated with explaining comment

Reply:
I left this one as it is as the recently introduced `offset` was never used and depreciation would only pollute the code. Is that OK?